fable-test-app/src/components/NextHoliday.tsx

```diff
@@ -23,6 +29,27 @@ const NextHoliday: React.FC<NextHolidayProps> = ({
     return Math.floor((holidayDate.getTime() - today.getTime()) / (1000 * 3600 * 24));
   };
 
+  // Get long version of date
+  const nextHolidayDate = (dateString: string) => {
+    const holidayDate = new Date(dateString);
+
+    return `${holidayDate.toLocaleString('default', { month: 'long' })} ${holidayDate.getDate()}`;
+  };
+
+  // Create formatted list of <abbr> elements for provinces
+  const formatObservedIn = () => {
```

Comment:
```suggestion
  const getObservedInProvinces = () => {
```
or something similar; `format*` typically means you pass it something it'll change but this one just simply returns a block of html